components/frontend/src/metric/MetricDebtParameters.js

```diff
@@ -74,7 +74,7 @@ function TechnicalDebtEndDate({ metric, metric_uuid, reload }) {
             requiredPermissions={[EDIT_REPORT_PERMISSION]}
             label={
                 <LabelWithDate
-                    date={Date(metric.debt_end_date)}
+                    date={metric.debt_end_date}
```

Comment:
I'm surprised that there's no change in test code needed?

Reply:
Good point. I'll add a unit test.